src/Event/Events/EventFromUDB2.php

```diff
@@ -201,4 +215,28 @@ public function getExternalId(): ?string
         $eventAsArray = $this->getEventAsArray();
         return $eventAsArray['location'][0]['label'][0]['@attributes']['externalid'] ?? null;
     }
+
+    private function getModeration(?string $wfstatus, ?string $lastUpdated): ?AbstractEvent
```

Comment:
Can these parameters be nullable if an `isset` is used before calling it?

Reply:
Indeed, fixed it.